docs/guides/ipfs.md

```diff
@@ -0,0 +1,110 @@
+# IPFS
+
+IPFS (InterPlanetary File System) is a suite of protocols for publishing data (files, directories, websites, etc.) in a decentralised fashion.
+For more info, see [What is IPFS](https://docs.ipfs.tech/concepts/what-is-ipfs/).
+
+There is an option to use some Lido interfaces via IPFS, for example Lido Ethereum Staking Widget.
+
+#### We use IPFS because:
+- IPFS has no single point of failure. The failure of a single or even multiple nodes in the network does not affect the functioning of the entire network.
+- IPFS is decentralized, which makes IPFS more resilient than traditional systems.
+- IPFS uses cryptographic hashes to verify the authenticity and integrity of files, making it difficult for malicious actors to affect files.
+
+## Address
+### What is a CID
+A content identifier, or CID, is a label used to point to material in IPFS. CIDs are based on the content’s cryptographic hash.
+Any difference in the content will produce a different CID.
+But CIDs won't match file hashes (checksums), because CID contains additional information that the hash does not.
+
+### IPFS HTTP Gateways
+An IPFS gateway is a web-based service that gets content from an IPFS network, and makes it available via HTTP protocol 
+that all web browsers understand.  
+A gateway address can look like this: `https://{CID}.ipfs.cf-ipfs.com`
+
+:::note
+Some browsers support IPFS natively and can access IPFS content without gateways, using canonical addressing like  
+`ipfs://{CID}/{optional path to resource}`  
+
+Also, there is the [IPFS Companion Browser Extension](https://docs.ipfs.tech/install/ipfs-companion), which enables support for IPFS in your browser. 
```

Comment:
This will not work without a locally running IPFS node
https://docs.ipfs.tech/install/ipfs-companion/#prerequisites

Not sure if this information is necessary for this page.